Lib/test/test_logging.py

```diff
@@ -4354,15 +4354,40 @@ def test_dict_arg(self):
         r.removeHandler(h)
         h.close()
 
+    @classmethod
```

Comment:
Why a class method rather than an instance method?

Reply:
It's the target for the child process.

Reply:
Actually I should probably rename it- it's not really checking the process name, just extracting it. The check is done in the main process.

Reply:
> It's the target for the child process.

Could it not be just a function?

Reply:
+1 for renaming it.

Reply:
It probably could be just a function, but then it's not right next to the test that uses it. Would you prefer that?

Reply:
or, do you mean staticmethod?

Reply:
staticmethod works (it can't be a nested function - it needs to be pickled to send to the child process)

Reply:
Then I think a staticmethod will be OK, with a comment indicating why for the casual reader :-) The above info would be useful to capture briefly in the code itself.

Reply:
Sure, added a comment. See if you can improve the wording :-)